src/Codebase/Infrastructure/Service/CodebaseService.php

```diff
@@ -34,6 +43,16 @@ public function __construct(SourceParser $sourceParser)
      */
     public function parseSource(CodebaseRequestDto $codebaseRequestDto): CodebaseSourceDto
     {
-        return $this->sourceParser->parseSource($codebaseRequestDto);
+        return $this->codeBaseReader->readCodeBase($codebaseRequestDto);
```

Comment:
Why parseSource method invokes readCodebase inside?

Reply:
Thanks! Renamed the method to `readCodeBase`